core/src/main/java/nu/yona/server/subscriptions/entities/Buddy.java

```diff
@@ -110,7 +109,12 @@ public UUID getUserId()
 
 	public User getUser()
 	{
-		return User.getRepository().findOne(userId);
+		return getUserIfExists().orElseThrow(() -> UserServiceException.notFoundById(userId));
```

Comment:
I would not expect a `UserServiceException` here. The `getUser` should probably be removed from this class otherwise.

Reply:
This is related to our naming scheme for exceptions. Most of our exceptions are named ``...ServiceException``, after the service that uses that exception _most_. We should rename our exceptions to functional names like `` UserException``.